src/platform/site-wide/mega-menu/containers/Main.jsx

```diff
@@ -48,10 +48,13 @@ export class Main extends React.Component {
   };
 
   updateCurrentSection = currentSection => {
-    recordEvent({
-      event: 'nav-header-second-level',
-      'nav-header-action': `Navigation - Header - Open Second Level - ${currentSection}`,
-    });
+    if (currentSection) {
```

Comment:
Nedie didn't want to track this when the currentSection was empty. It was previously recording these when currentSection was empty. It was empty on window resize to reset the MegaMenu and also when clicking the back button on mobile in the MegaMenu.